WAW.API/Chat/Domain/Repositories/IChatRoomRepository.cs

```diff
@@ -0,0 +1,11 @@
+using WAW.API.Chat.Domain.Models;
+
+namespace WAW.API.Chat.Domain.Repositories;
+
+public interface IChatRoomRepository {
+  Task<IEnumerable<ChatRoom>> ListAll();
+  Task Add(ChatRoom chatRoom);
+  Task<ChatRoom?> FindById(long id);
```

Comment:
I think there should also be a way to find a chat room by participant. So that if a `User` is found in the list of participants, it is included in the response, and it should return all the chat rooms that match.

The service and controllers should also have a way to get this information.

Reply:
I think the same but that method maybe can be implemented in the next controllers with relationships `UserChatRoomController`